hack/verify-generated-crd-code.sh

```diff
@@ -19,7 +19,7 @@ HACK_DIR=$(dirname "${BASH_SOURCE}")
 ${HACK_DIR}/update-3generated-crd-code.sh
 
 # ensure no changes to generated CRDs
-if ! git diff --exit-code config/crd/v1/crds/crds.go >/dev/null; then
+if [! git diff --exit-code config/crd/v1/crds/crds.go >/dev/null] || [! git diff --exit-code config/crd/v1alpha1/crds/crds.go >/dev/null]; then
```

Comment:
This can be simplified.
```suggestion
if [! git diff --exit-code config/crd/v1/crds/crds.go config/crd/v1alpha1/crds/crds.go >/dev/null]; then
```

Reply:
Thanks, let me change this later.

Reply:
Done